app-frontend/packs/router/index.js

```diff
@@ -0,0 +1,23 @@
+import { createRouter, createWebHistory } from 'vue-router'
+import HomePage from '../components/homePage/HomePage.vue'
+import EditPage from '../components/editPage/EditPage.vue'
+
+const routes = [
+    {
+        path: '/',
+        name: 'HomePage',
+        component: HomePage
+    },
+    {
+        path: '/Edit',
```

Comment:
пути обычно в ловер кейсе

```suggestion
        path: '/edit',
```

Reply:
done